Modules/CalcPerform.lua

```diff
@@ -618,18 +618,35 @@ function calcs.perform(env)
 		if activeSkill.activeEffect.grantedEffect.name == "Summon Skeletons" then
 			local limit = env.player.mainSkill.skillModList:Sum("BASE", env.player.mainSkill.skillCfg, "ActiveSkeletonLimit")
 			output.ActiveSkeletonLimit = m_max(limit, output.ActiveSkeletonLimit or 0)
-		end
-		if activeSkill.activeEffect.grantedEffect.name == "Raise Zombie" then
+		elseif activeSkill.activeEffect.grantedEffect.name == "Raise Zombie" then
 			local limit = env.player.mainSkill.skillModList:Sum("BASE", env.player.mainSkill.skillCfg, "ActiveZombieLimit")
 			output.ActiveZombieLimit = m_max(limit, output.ActiveZombieLimit or 0)
-		end
-		if activeSkill.activeEffect.grantedEffect.name == "Summon Raging Spirit" then
+		elseif activeSkill.activeEffect.grantedEffect.name == "Summon Raging Spirit" then
 			local limit = env.player.mainSkill.skillModList:Sum("BASE", env.player.mainSkill.skillCfg, "ActiveRagingSpiritLimit")
 			output.ActiveRagingSpiritLimit = m_max(limit, output.ActiveRagingSpiritLimit or 0)
-		end
-		if activeSkill.activeEffect.grantedEffect.name == "Raise Spectre" then
+		elseif activeSkill.activeEffect.grantedEffect.name == "Raise Spectre" then
 			local limit = env.player.mainSkill.skillModList:Sum("BASE", env.player.mainSkill.skillCfg, "ActiveSpectreLimit")
 			output.ActiveSpectreLimit = m_max(limit, output.ActiveSpectreLimit or 0)
+		elseif activeSkill.activeEffect.grantedEffect.name == "Enduring Cry" then
+			local heal_over_1_sec = activeSkill.skillModList:Sum("BASE", env.player.mainSkill.skillCfg, "EnduringCryLifeRegen")
+			local resist_all_per_endurance = activeSkill.skillModList:Sum("BASE", env.player.mainSkill.skillCfg, "EnduringCryElementalResist")
+			local pdr_per_endurance = activeSkill.skillModList:Sum("BASE", env.player.mainSkill.skillCfg, "EnduringCryPhysicalDamageReduction")
+			local duration_inc = env.player.mainSkill.skillModList:Sum("INC", env.player.mainSkill.skillCfg, "Duration")
+			local full_duration = activeSkill.activeEffect.grantedEffectLevel.duration * (1 + duration_inc / 100)
+			local cdr = env.player.mainSkill.skillModList:Sum("INC", env.player.mainSkill.skillCfg, "CooldownRecovery")
+			local actual_cooldown = activeSkill.activeEffect.grantedEffectLevel.cooldown / (1 + cdr / 100)
+			local uptime = m_min(full_duration / actual_cooldown, 1)
+			modDB:NewMod("LifeRegen", "BASE", heal_over_1_sec * uptime, "Enduring Cry")
```

Comment:
It would be nice if this Life Regen used the average and burst system that is in place. You do that by simply adding two mods here. One with a condition for "LifeRegenBurstAvg"(divide the value based on how often that 1 second buff will be up. it won't be able to use the local uptime you calculate right above this). Then a second mod with a condition "LifeRegenBurstFull"(just give the full regen).

Reply:
I did this in upcoming push... but why does Minimum exist for Life Regen if nothing sets the flag for it?

Reply:
its just uh, not average or maximum, if you dont want either of those 2